.github/workflows/build.yml

```diff
@@ -188,8 +227,72 @@ jobs:
             docker stop $(docker ps -f name=platform-explorer-api -q)
             docker stop $(docker ps -f name=platform-explorer-indexer -q)
             docker rm platform-explorer-api platform-explorer-indexer
-            docker run --rm --env-file api.env ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:api npm run db:drop
-            docker run --rm --env-file api.env -v ./platform-explorer/packages/indexer/migrations:/migrations owl352/refinery_cli_docker refinery migrate -e DATABASE_URL -p /migrations
             docker run -d -p 3005:3005 --restart always --env-file api.env --name platform-explorer-api ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:api
             sleep 3
             docker run -d --env-file api.env --restart always --name platform-explorer-indexer ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:indexer ./indexer
+            
+  create_tag:
+    runs-on: ubuntu-latest
+    needs: [build_api, build_indexer, test_data_contract, test_api, test_frontend]
+    
+    if: github.event_name != 'pull_request' || github.event.pull_request.merged == true
```

Comment:
I think you need to handle tag pushes instead of pull requests in the top of the file

Reply:
✅